app/views/targets/index.json.jbuilder

```diff
@@ -0,0 +1,9 @@
+json.array! @all_targets do |target|
+  json.set! :topic, target.topic
```

Comment:
Ideally these lines should be a partial but if you don't use it from any view (like show, etc.) then I guess it's fine. 
Also, try to format the lines so it looks like this:
```
json.set! :topic,    target.topic
json.set! :length,   target.length
json.set! :latitude, target.latitude
```